web/server.js

```diff
@@ -158,9 +159,28 @@ app.get('/', (req, res) => {
     }
     const { locale } = res;
     const fullUrl = 'https://www.electricitymap.org' + req.originalUrl;
+
+    // basic auth and time machine cookie if we're in the back in time container
+    if (process.env['BASIC_AUTH_USERNAME;']){
+      let user = auth(req);
+      let authorized = false;
+      if (user){
+        let ix = process.env['BASIC_AUTH_USERNAMES'].split(';').indexOf(user.name);
```

Comment:
You can be more explicit by naming a variable `hasMatchingUserName` and directly do the `!== -1` comparaison here.